test/e2e/readme.md

```diff
@@ -2,12 +2,14 @@
 
 Celestia uses the [knuu](https://github.com/celestiaorg/knuu) framework to orchestrate clusters of nodes in a network for end to end testing. This relies on Docker and a kubeconfig (in `~/.kube/config`) to access the Kubernetes cluster.
 
+End to end tests pull docker images from ghcr.io/celestiaorg/celestia-app. These are automatically published when tagging a new release or when opening a pull request. If you wish to manually test a specific commit, you can manually publish the image by first running `make build-ghcr-docker` (from the root directory) and then running `make publish-ghcr-docker`. You must have permission to push to the ghcr.io/celestiaorg/celestia-app repository.
```

Comment:
> or when opening a pull request

[question] if Github CI pushes a new docker image for every commit on PRs then it seems easier / faster to push a commit and let Github CI push the docker image. Out of curiosity, is there a big time delay when Github CI does it that developers can avoid by doing this locally?

Reply:
I guess my main reason for running this is that I don't want to open a PR just to publish something on docker. It's probably slightly quicker locally plus you know exactly when it has been published and can be pulled by the knuu tests

Reply:
Makes sense, can resolve.